icu4c/source/tools/pkgdata/pkgdata.cpp

```diff
@@ -2203,21 +2208,18 @@ static void loadLists(UPKGOptions *o, UErrorCode *status)
         }
     }
 
-    if(buf[strlen(buf)-1]=='\n')
-    {
-        buf[strlen(buf)-1]=0;
-    }
-
-    if(buf[0] == 0)
-    {
-        fprintf(stderr, "%s: icu-config: invalid response from icu-config (fix PATH or use -O option)\n", progname);
+    if (!buf[0]) {
+        fprintf(stderr, "%s: invalid response from pkg-config or icu-config\n", progname);
```

Comment:
(Ah, i lost my code I was writing… ) 

Basically, I was suggesting to remember which of `pkgconfigCmd` or `icuconfigCmd` was last used .And then print out  something like this:

> "pkgdata: Could not locate config file. Could not parse the results from 'icu-config --incpkgdatafile'. Check paths or use the -O option to specify a pkgdata.inc file."

Reply:
Sure. I've updated the PR with both changes.